arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -249,9 +244,8 @@
             }
           }
         },
-        "deprecated": false,
         "x-ms-pageable": {
-          "nextLinkName": null
+          "nextLinkName": "nextLink"

```

Comment:
Just adding this thing will not be enough. You need to modify RegistryListResult type as well. It should look like this:
```
"RegistryListResult": {
      "description": "The result of listing container registries.",
      "properties": {
        "value": {
          "type": "array",
          "items": {
            "$ref": "#/definitions/Registry"
          }
        },
        "nextLink": {
          "type": "string",
          "description": "Gets or sets the URL to get the next set of results."
        }
      }
    }
```